src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random number = new Random();
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private final int lowerBound = 1;
+    private final int upperBound = 101;
```

Comment:
The lower and upper bounds should be constants. According to Java naming conventions, constants should be in uppercase (e.g., LOWER_BOUND, UPPER_BOUND).